demo/OrderCloud.DemoWebApi.Tests/TestFramework.cs

```diff
@@ -21,6 +21,7 @@ public static IFlurlClient Client
 			get
 			{
 				var server = new TestServer(CatalystWebHostBuilder.CreateWebHostBuilder<TestStartup, AppSettings>(new string[] { }));
+				server.AllowSynchronousIO = true; // for webhook tests
```

Comment:
Is it possible to refactor the webhook auth code so we don't need this? If not, I think we need to document the fact that this needs to be set to true by the implementor and what it means.  

Just for reference here's the comment I added in SEB:

> // false became the default in asp.net core 3.0 to combat application hangs
                // however we're using synchronous APIs when validating webhook hash
                // specifically ComputeHash will trigger an error here
                // TODO: figure out how to compute the hash in an async manner so we can remove this